app/models/member.rb

```diff
@@ -1,8 +1,21 @@
 class Member < ApplicationRecord
   AVERAGE_WEEKS_PER_MONTH = 4.33
   MONTHS_PER_YEAR = 12
+  PAYMENT_INTERVALS = [
```

Comment:
nice! We should maybe be doing those for more fields in general. Hard to keep track of validations when they're all in the steps! Although are we worried about having duplicate validations and them getting out of sync?